actions/user_actions.jsx

```diff
@@ -284,9 +284,9 @@ export function loadProfilesForGroupChannels(groupChannels) {
     };
 }
 
-export function loadProfilesForSidebar() {
-    loadProfilesForDM();
-    loadProfilesForGM();
+export async function loadProfilesForSidebar() {
```

Comment:
Is there a reason to not use Promise.all here? If I understand correctly this doesn't needs to run sequentially.

Reply:
Nope. Good call, I changed it to `Promise.all`